lethe.js

```diff
@@ -349,7 +396,18 @@ client.on('message', m => {
           formattedList += formattedVideo;
         }
       });
-      formattedList += `\n**Remaining play time:** ${Util.formatTime(overallTime)} minutes.`;
+	  var hoursOf = Math.floor(overallTime / 60 / 60);
```

Comment:
Could you make this code into a separate function so 1) the commands are less cluttered and 2) there's no duplicate code (same function should be used in `list all`)?